mongo/mongo.go

```diff
@@ -469,13 +500,12 @@ func ensureServer(args EnsureServerParams, mongoKernelTweaks map[string]string)
 	}
 	logVersion(mongoPath)
 
-	if err := UpdateSSLKey(args.DataDir, args.Cert, args.PrivateKey); err != nil {
-		return zeroVersion, errors.Trace(err)
-	}
-
-	err = utils.AtomicWriteFile(sharedSecretPath(args.DataDir), []byte(args.SharedSecret), 0600)
-	if err != nil {
-		return zeroVersion, errors.Errorf("cannot write mongod shared secret: %v", err)
+	oplogSize := args.OplogSize
```

Comment:
The original var was called oplogSizeMB to convey to the user the units.
There's no need to change that var name

Reply:
Ah, I was shadowing the [field name](https://github.com/juju/juju/blob/484c173746b4560d724f099aa60e6c516ccf79ac/mongo/mongo.go#L449) to make it obvious that they're connected.

Are you happy for the local variable to be renamed back to `oplogSizeMB` or would you like that change to be made further upstream where the user is more likely to encounter it?